cirq-core/cirq/transformers/synchronize_terminal_measurements.py

```diff
@@ -55,7 +55,7 @@ def find_terminal_measurements(
         seen_control_keys |= protocols.control_keys(moment)
         if not open_qubits:
             break
-    return terminal_measurements
+    return list(set(terminal_measurements))
```

Comment:
Perhaps we should convert the local `terminal_measurement` to a set to remove one list --> set conversion.

Do we care about the returned list order?

If not we can add a comment to the docstring.

Reply:
done